integrationTests/mock/heartbeatStorerStub.go

```diff
@@ -3,15 +3,15 @@ package mock
 import (
 	"time"
 
-	"github.com/ElrondNetwork/elrond-go/node/heartbeat"
+	"github.com/ElrondNetwork/elrond-go/heartbeat/data"
 )
 
 // HeartbeatStorerStub -
 type HeartbeatStorerStub struct {
 	LoadGenesisTimeCalled   func() (time.Time, error)
 	UpdateGenesisTimeCalled func(genesisTime time.Time) error
-	LoadHbmiDTOCalled       func(pubKey string) (*heartbeat.HeartbeatDTO, error)
-	SavePubkeyDataCalled    func(pubkey []byte, heartbeat *heartbeat.HeartbeatDTO) error
+	LoadHbmiDTOCalled       func(pubKey string) (*data.HeartbeatDTO, error)
```

Comment:
I think Hbmi could be replaced with Heartbeat everywhere in all these files

Reply:
That's enough!! Cheating bastards.
The price keeps falling. People don't trust you. ERD doesn't buy. What do you boys do for a living?

Reply:
done